libs/web/providers/src/CENNZExtensionProvider.tsx

```diff
@@ -73,19 +67,15 @@ export const CENNZExtensionProvider: FC<CENNZExtensionProviderProps> = ({
 	}, [appName, module]);
 
 	useEffect(() => {
-		if (!module || !browser?.name) return;
+		if (!module || !runtimeMode || runtimeMode === "ReadOnly") return;
```

Comment:
just a question here, why do need `!runtimeMode` here?

Reply:
useEffect was picking up the default state of ReadWrite too quickly on Safari